requirements-app.txt

```diff
@@ -1,7 +1,7 @@
 # Run `make freeze-requirements` to update requirements.txt
 # with package version changes made in requirements-app.txt
 
-aws-wsgi==0.2.7
+apig-wsgi==2.13.0
```

Comment:
This is a fairly big change to switch library. It seems more up to date though compared to `aws-wsgi`. Should we have plans for testing or you are confident we can deploy this to production?

Reply:
Confident, but I'd still like to give things a thorough test in Staging when this merges.  

The problem with the `aws-wsgi` module is that it was no longer supported and missing a lot of features that AWS has added for Lambda proxy events (like the multi-value params and support for `v2.0` of the event payload).

I found `apig-wsgi` yesterday while trying to solve problems with creating the Notify admin PR review env.